### ISSUE TITLE
Support private git repos in workload onboarding

### ISSUE THREAD

**Issue by jpellizzari** [fluxcd/webui#14]
We assume here the workload comes from a public repo, but I think in most cases it comes from a private repo. I would add `flux create secret git <SSH secret>` and `flux create source git --secretRef=<SSH secret>`.

_Originally posted by @stefanprodan in https://github.com/fluxcd/webui/pull/12#discussion_r594516807_

**Comment by gonzaloserrano** [comment]
Yep, I bet most cases use GitHub private repos. I had to read https://github.com/podtato-head/podtato-head/blob/main/delivery/flux/README.md in order to understand the whole thing.
